subworkflows/local/rna_fusions_jaffal.nf

```diff
@@ -0,0 +1,34 @@
+include { GET_JAFFAL_REF } from '../../modules/local/get_jaffal_ref'
+include { UNTAR          } from '../../modules/nf-core/modules/untar/main'
+include { JAFFAL         } from '../../modules/local/jaffal'
+
+workflow RNA_FUSIONS_JAFFAL {
+    take:
+    ch_sample
+    jaffal_ref_dir
+
+    main:
+
+    if (jaffal_ref_dir) {
+        ch_jaffal_ref_dir = file(params.jaffal_ref_dir, checkIfExists: true)
+    } else {
+        GET_JAFFAL_REF()
+        UNTAR( GET_JAFFAL_REF.out.ch_jaffal_ref )
+        ch_jaffal_ref_dir = UNTAR.out.untar
+    }
+    ch_jaffal_ref_dir.view()
+
+    ch_sample
+        .map { it -> [ it[0], it[6] ]}
+        .set { ch_jaffal_input }
+
+    /*
+    * Align current signals to reference with Nanopolish
+    */
+    JAFFAL( ch_jaffal_input, ch_jaffal_ref_dir )
+
+//    emit:
```

Comment:
I don't know much about this analysis but I just wanted to check these were commented on purpose

Reply:
I forgot to clean up. These are not related to the RNA fusion detection analysis